dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -41,7 +41,6 @@ RUN apt-get -y install build-essential wget libssl-dev openssl supervisor && \
     python3 -m sonic_ax_impl install && \
     python3 -m pip uninstall -y pip setuptools && \
     /bin/bash -c "rm -rf /usr/lib/python3.5/{unittest,lib2to3,tkinter,idlelib,email,test}" && \
-    apt-get -y purge build-essential wget libssl-dev openssl && \
```

Comment:
I just double checked it and it is not openssl but wget. sonic-config-engine->python-netaddr->ieee-data->wget.

However, I still feel that we should remove all the purges since they only save ~10M space but cause a lot of potential confusion and difficulties in debugging.